zerver/openapi/python_examples.py

```diff
@@ -216,6 +216,79 @@ def get_members(client):
         else:
             assert member.get('profile_data', None) is not None
 
+@openapi_test_function("/users/{user_id}:get")
+def get_single_user(client):
+    # type: (Client) -> None
+
+    # {code_example|start}
+    # Get user data of a user
+    user_id = 8
+    url = 'users/' + str(user_id)
+    result = client.call_endpoint(
+        url=url,
+        method='GET'
+    )
+    # {code_example|end}
+    validate_against_openapi_schema(result, '/users/{user_id}', 'get', '200')
+
+    # {code_example|start}
+    # You may pass the `client_gravatar` query parameter as follows:
+    result = client.call_endpoint(
+        url=url,
+        method='GET',
+        request={'client_gravatar': True}
+    )
+    # {code_example|end}
+
+    validate_against_openapi_schema(result, '/users/{user_id}', 'get', '200')
+    assert result['members'][0]['avatar_url'] is None
+
+    # {code_example|start}
+    # You may pass the `include_custom_profile_fields` query parameter as follows:
+    result = client.call_endpoint(
+        url=url,
+        method='GET',
+        request={'include_custom_profile_fields': True}
+    )
+    # {code_example|end}
+
+    validate_against_openapi_schema(result, '/users/{user_id}', 'get', '200')
+
+@openapi_test_function("/users/{user_id}:delete")
+def deactivate_user(client):
+    # type: (Client) -> None
+
+    # {code_example|start}
+    # Get user data of a user
+    user_id = 8
+    url = 'users/' + str(user_id)
+    result = client.call_endpoint(
+        url=url,
+        method='DELETE',
+    )
+    # {code_example|end}
+    validate_against_openapi_schema(result, '/users/{user_id}', 'delete', '200')
+
+@openapi_test_function("/users/{user_id}:patch")
+def update_user(client):
+    # type: (Client) -> None
+
+    # {code_example|start}
+    # Get user data of a user
+    user_id = 8
+    profile_data = [{
+                    'id': 9,
+                    'value': 'some data'
+                    }]
```

Comment:
I would start with an example for changing a user's name, and then have a second example for patching custom profile fields (which are a more complex corner case, which we'll eventually need further documentation to explain what the IDs are)